tt_metal/programming_examples/eltwise_sfpu/eltwise_sfpu.cpp

```diff
@@ -77,13 +77,13 @@ int main(int argc, char **argv) {
          */
         KernelHandle unary_reader_kernel_id = CreateKernel(
             program,
-            "tt_metal/kernels/dataflow/reader_unary.cpp",
+            "ttnn/cpp/ttnn/operations/examples/example/device/kernels/dataflow/reader_unary.cpp",
```

Comment:
I thought tt_metal was not supposed to reference ttnn. Has that changed or are there carve outs?

Reply:
Same concern here. Why did we need to change the kernel being used to a ttnn kernel? If we can't reuse the old one then should duplicate the kernel into tt_metal dir, ex `tt_metal/programming_examples/eltwise_sfpu/kernels/dataflow` if it's specific for this test

Reply:
thought I caught this.  done